torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1324,8 +1330,12 @@ def codegen_alloc_from_pool(self, name, offset, dtype, shape, stride) -> str:
                 pexpr(offset),  # bytes not numel
                 self.codegen_dtype(dtype),
                 str(len(shape)),
-                self.codegen_int_array_var(size, self.wrapper_call),
-                self.codegen_int_array_var(stride, self.wrapper_call),
+                self.codegen_int_array_var(
+                    size, self.wrapper_call, graph=self.get_codegened_graph()
```

Comment:
`codegen_int_array_var` relies on @lru_cache to cache on the arguments. The reason I've added the `graph` argument is to include the codegened graph into the cache key, hence make caching per codegened graph (or scope).

I don't think we can make the `self.get_codegened_graph()` the default value of the `graph` argument, as this will run into default being captured on the first pass (and not be changed in further calls). Therefore, I'm passing `self.get_codegened_graph()` from every call site to appropriately reflect the codegen graph at that call site in the arguments of `codegen_int_array_var`. Does this make sense, or do you mean something else?